#! /bin/sh /usr/share/dpatch/dpatch-run
## quiet-retrieve.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: when no address defined for the web root, ask to reload the server, 
## DP: and don't type an empty address. the code has been taken from install_amp
## DP: comited upstream at 3981, to be removed for 2.2.2 or 2.3

@DPATCH@
diff -urNad upstream~/amp_conf/bin/retrieve_conf upstream/amp_conf/bin/retrieve_conf
--- upstream~/amp_conf/bin/retrieve_conf	2007-01-04 02:48:29.000000000 +0200
+++ upstream/amp_conf/bin/retrieve_conf	2007-05-14 16:15:19.000000000 +0300
@@ -431,5 +431,14 @@
 	
 // **** Set reload flag for AMP admin
 needreload();
-out("Please Reload Asterisk by visiting http://".$amp_conf["AMPWEBADDRESS"]."/admin");
+if ($amp_conf["AMPWEBADDRESS"])
+{
+        out("Please update your modules and reload Asterisk by visiting http://".$amp_conf["AMPWEBADDRESS"]."/admin");
+}
+else
+{
+        out("Please update your modules and reload Asterisk by browsing to your server.");
+}
+
+
 ?>
